apps/web/src/actions/procedures/index.ts

```diff
@@ -16,11 +16,11 @@ export const authProcedure = createServerActionProcedure().handler(async () => {
 export const withProject = createServerActionProcedure(authProcedure)
   .input(z.object({ projectId: z.number() }))
   .handler(async ({ input, ctx }) => {
+    const { workspace } = ctx
+    const projectScope = new ProjectsRepository(workspace.id)
     const project = (
-      await findProject({
-        projectId: input.projectId,
-        workspaceId: ctx.workspace.id,
-      })
+      await projectScope.getProjectById(input.projectId)
```

Comment:
Would be nice that this is a class call it `find` when is by ID

Reply:
🤷🏼 can change if we prefer it